css/mistica-common.css

```diff
@@ -133,3 +249,331 @@
+}
+
+.mistica-snap-card {
+    padding: 16px;
+}
+
+.mistica-display-card {
+    padding: 24px;
+    border-radius: var(--mistica-border-radius-legacyDisplay);
+    justify-content: flex-end;
+}
+
+.mistica-naked-card,
+.mistica-small-naked-card {
+    padding-right: 16px;
+}
+
+.mistica-aspect-ratio {
+    display: flex;
+    flex: 1;
+}
+
+.mistica-aspect-ratio::before {
+    content: '';
+    padding-bottom: calc(100% / (var(--aspect-ratio)));
+}
+
+.mistica-card .mistica-card__media {
+    margin: -24px -16px 16px -16px;
+    aspect-ratio: 16 / 9;
+    object-fit: cover;
+    width: calc(100% + 32px);
+    max-width: initial; /* Movistar AEM has a default max-width: 100% for any img tag */
+}
+
+.mistica-naked-card .mistica-card__media,
+.mistica-small-naked-card .mistica-card__media {
+    aspect-ratio: 16 / 9;
+    object-fit: cover;
+    width: 100%;
+    border-radius: var(--mistica-border-radius-container);
+    margin-bottom: 16px;
+}
+
+.mistica-display-card .mistica-card__media {
+    object-fit: cover;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+}
+
+.mistica-display-card:has(.mistica-card__media) {
+    isolation: isolate;
+    padding-top: 40px;
+    --mistica-card-text-shadow: 0 0 16px rgba(0, 0, 0, 0.4);
+    --mistica-card-text-color: var(--mistica-color-textPrimaryInverse);
+    --mistica-card-description-text-color: var(--mistica-color-textPrimaryInverse);
+}
+
+.mistica-display-card:has(.mistica-card__media):after {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: var(--mistica-color-cardContentOverlay);
+    z-index: -1;
+}
+
+.mistica-card__pretitle {
+    font-size: var(--mistica-font-size-2);
+    line-height: var(--mistica-line-height-2);
+    font-weight: var(--mistica-font-weight-2);
+    text-shadow: var(--mistica-card-text-shadow);
+}
+
+[class^='mistica-tag'] + .mistica-card__pretitle {
+    padding-top: 8px;
+}
+
+.mistica-display-card [class^='mistica-tag'] + .mistica-card__pretitle {
+    padding-top: 16px;
+}
+
+.mistica-card__title {
+    font-size: var(--mistica-font-size-cardTitle);
+    line-height: var(--mistica-line-height-cardTitle);
+    font-weight: var(--mistica-font-weight-cardTitle);
+    text-shadow: var(--mistica-card-text-shadow);
+}
+
+.mistica-snap-card .mistica-card__title,
+.mistica-small-naked-card .mistica-card__title {
+    font-size: var(--mistica-font-size-2);
+    line-height: var(--mistica-line-height-2);
+}
+
+.mistica-display-card .mistica-card__title {
+    font-size: var(--mistica-font-size-6);
+    line-height: var(--mistica-line-height-6);
+    font-weight: var(--mistica-font-weight-6);
+}
+
+[class^='mistica-tag'] + .mistica-card__title {
+    padding-top: 8px;
+}
+
+.mistica-display-card [class^='mistica-tag'] + .mistica-card__title {
+    padding-top: 16px;
+}
+
+.mistica-card__pretitle + .mistica-card__title {
```

Comment:
.mistica-card__title should use a custom token for font-weight `cardTitle`

https://github.com/Telefonica/mistica-web/blob/6061b676a991db6a0dfd02f84ba29f360a75e627/src/card.tsx#L480

Reply:
I think I'm already applying that weight